src/lib/Resources/settings/storage_engines/legacy/content_type.yml

```
@@ -59,3 +61,17 @@ services:
 
     ezpublish.spi.persistence.legacy.content_type.handler:
         alias: ezpublish.spi.persistence.legacy.content_type.handler.caching
+
+    Ibexa\Core\Persistence\Legacy\Content\Type\StorageRegistryInterface:
+        alias: Ibexa\Core\Persistence\Legacy\Content\Type\StorageRegistry
+
+    Ibexa\Core\Persistence\Legacy\Content\Type\StorageRegistry:
+        arguments:
+            $storages: !tagged_iterator { tag: ibexa.field_type.external_constraints_storage, index_by: alias }
```

Can we do a bit more grouping here for the tag name:
```suggestion
            $storages: !tagged_iterator { tag: ibexa.field_type.storage.external.constraints, index_by: alias }
```
?
POV ping @Steveb-p @mikadamczyk

---

Just make sure it's consistent with tags related to external storage for values

* ezplatform.field_type.external_storage_handler
* ezplatform.field_type.external_storage_handler.gateway

---

`ibexa.field_type.storage.external.constraints.handler` was discussed.

`ibexa.field_type.storage.external.handler`
`ibexa.field_type.storage.external.handler.gateway` for others.